buffer/src/main/java/io/netty/buffer/ByteBufHolder.java

```diff
@@ -32,6 +32,20 @@
      */
     ByteBufHolder copy();
 
+    /**
+     * Slices this {@link ByteBufHolder}. Be aware that this will not automatically call {@link #retain()}.
+     *
+     * @see ByteBuf#slice()
+     */
+    ByteBufHolder slice();
```

Comment:
Breaking change, not suitable for 4.1.

Reply:
This is for Netty 5 as said in the linked issue.